avocado/core/loader.py

```diff
@@ -207,16 +223,10 @@ def get_base_keywords(self):
         return base_path
 
     def get_type_label_mapping(self):
-        mapping = {}
-        for loader_plugin in self._initialized_plugins:
-            mapping.update(loader_plugin.get_type_label_mapping())
-        return mapping
```

Comment:
That's quite obvious, but does not really answer the question.
This code was here in your v2 (#1926) and was moved in v3 and v4. Since I already pointed this out in the v3 and you kept it in v4 (actually it's worst in v4, with a new private function), I'd like to understand the reason for that.